.github/workflows/test.yml

```
@@ -311,6 +311,73 @@ jobs:
           path: frontend/test-results/
           retention-days: 7
 
+  # ── Self-hosted benchmark mirrors ──────────────────────────────────
+  lint-audit-self-hosted:
+    name: Lint & Security Audit (Self-Hosted)
+    runs-on: [self-hosted, ci]
```

actionlint (1.7.10)

[error] label "ci" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2025", "windows-2022", "windows-11-arm", "ubuntu-slim", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-24.04-arm", "ubuntu-22.04", "ubuntu-22.04-arm", "macos-latest", "macos-latest-xlarge", "macos-latest-large", "macos-26-xlarge", "macos-26", "macos-15-intel", "macos-15-xlarge", "macos-15-large", "macos-15", "macos-14-xlarge", "macos-14-large", "macos-14", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows", "sonar". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)